packages/optimizer/lib/fetchRuntimeParameters.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'mode strict';
+
+const validatorRules = require('@ampproject/toolbox-validator-rules');
+const {AMP_CACHE_HOST, AMP_RUNTIME_CSS_PATH, appendRuntimeVersion} = require('./AmpConstants.js');
+
+/**
+ * Creates the runtime parameters used by the transformers. Runtime parameters might
+ * change at runtime with new AMP releases.
+ *
+ * @param {Object} config - the AMP Optimizer config
+ * @param {Object} customRuntimeParameters - user defined runtime parameters
+ */
```

Comment:
You might consider adding a `@returns` to quickly identify the parameters that are typically returned. It's a bit difficult to identify them all by reading through the code. Similarly, in the description, you might mention which parameters are simply validated, which parameters are fetched and verified via network, and the rest pass-through untouched.

Reply:
Done